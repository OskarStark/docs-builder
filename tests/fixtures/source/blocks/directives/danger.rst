
.. danger::

    This message is about security risk or data integrity threat.
