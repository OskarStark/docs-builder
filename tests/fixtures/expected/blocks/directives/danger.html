<div class="admonition admonition-danger ">
    <p class="admonition-title">
       <svg xmlns="http://www.w3.org/2000/svg" fill="none" width="24" height="24" viewBox="0 0 24 24" stroke="currentColor"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M12 8v4m0 4h.01M21 12a9 9 0 11-18 0 9 9 0 0118 0z" /></svg>
                <span>Danger</span>
    </p><p>This message is about security risk or data integrity threat.</p>
</div>
